### PR TITLE
ddccontrol

### PR DESCRIPTION
**Pull request by Gottox**
This PR adds the ddccontrol package that I find handy to use the kvm feature of my dell monitor.

### CODE REVIEW

File: srcpkgs/ddccontrol/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'ddccontrol'
+pkgname=ddccontrol
+version=0.4.4
+revision=1
+build_style=gnu-configure
+configure_args=" --disable-gnome --disable-gnome-applet --disable-dependency-tracking"
+hostmakedepends="automake tar libtool"
```
Comment (fosslinux) [inline srcpkgs/ddccontrol/template:7]: `gettext-devel` is needed here as well? instead? as/of `makedepends`

Comment (Piraty) [reply srcpkgs/ddccontrol/template:7]: yes. https://travis-ci.org/github/void-linux/void-packages/jobs/714610166#L835

**Comment by the-maldridge** [comment]
@Gottox can you rebase so we can have github actions CI results for this PR?

**Comment by Piraty** [comment]
this at least needs to be restricted to `archs="i686* x86_64*"` for musl, since [src/ddcpci/sis.c](https://github.com/ddccontrol/ddccontrol/blob/f3d003f63aaa0407d3147b04848e0722f78695d5/src/ddcpci/sis.c#L68) uses `inb`/`outb` which musl only defines for x86* (`arch/{i386,x32,x86_64}/bits/io.h`)

**Comment by Gottox** [comment]
shouldn't it be `archs="i686-musl x86_64-musl ~*-musl"` then?

**Comment by Gottox** [comment]
@the-maldridge rebase

**Comment by ericonr** [comment]
@Piraty I don't think this is functional on ARM glibc either...

Their `outb` and `inb` are purely compatibility symbols:

```void
outb (unsigned char b, unsigned long int port)
{
}
compat_symbol (libc, outb, outb, GLIBC_2_4);
```

**Comment by Gottox** [comment]
From the documentations it looks like those are gcc builtins:

```
man outb
...
You must compile with -O or -O2 or similar.  The functions are defined
as inline macros, and will not be substituted in without optimization
enabled, causing unresolved references at link time.
...
```

**Comment by ericonr** [comment]
@Gottox I believe you need to rebase. The autopoint issue was fixed on the gettext side, iirc.

File: srcpkgs/ddccontrol/template (hunk 1 above)
Comment (ericonr) [reply srcpkgs/ddccontrol/template:7]: @Gottox I got confused, autopoint issues were something else. Just need to add it to hostmakedepends.

**Comment by dkwo** [comment]
Does this package provide anything that `ddcutil` does not?

**Comment by Gottox** [comment]
No need for ddccontrol, we already ship ddcutil.

**Pull request closed without merging**